### DISCUSSION TITLE
Feature request: Option to disable the "floating dialogs"

### DISCUSSION THREAD

**Discussion by loicb24** [burkybang/SuiteAdvanced-Feedback#212] [Ideas]
Hi, 

I sometimes work from my laptop's smaller screen and the fact that the dialogs are "floating" is a bit annoying, as it hides the standard buttons of NetSuite, that I constantly have to close. 

![image](https://github.com/user-attachments/assets/ebac1bbb-97a0-4ac3-b099-0c279f05f7b1)

I wish I had an option in the SuiteAdvanced options to enable/disable the "floating dialog" (showing them as the standard dialogs) : 

![image](https://github.com/user-attachments/assets/ad7dc633-028b-4d7e-92f5-4d5766e08c26)

Hope that this is a small option you can introduce in a future release.

Loïc

**Comment by burkybang** [comment]
I'll consider this. In the meantime, you can manually close them or set them to auto close under the Timings section in Settings.

![image](https://github.com/user-attachments/assets/2211e83f-94a0-40b1-89cc-c81267c5df0f)
